conf/modules.config

```diff
@@ -412,6 +412,14 @@ process {
         ]
     }
 
+    withName: '.*CALL_SNV:CALL_SNV_SENTIEON:BCFTOOLS_FILTER' {
+        publishDir = [
+            path: { "${params.outdir}/bcftools_filter" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
```

Comment:
Add the same section for the second alias of bcftools_filter that you make in the subworkflow with the following arguments:
ext.args = '-i "'FILTER=\"PASS\"'" -m x'  #We need to make sure later if strings get parsed correctly.